src/components/selector-dropdown/SelectorDropdown.js

```diff
@@ -256,7 +256,11 @@ class SelectorDropdown extends React.Component<Props, State> {
         }
 
         const list = (
-            <ul className="overlay" id={listboxID} role="listbox">
+            <ul
+                className={classNames('overlay', overlayTitle ? overlayTitle.toLowerCase() : '')}
```

Comment:
See [docs](https://www.npmjs.com/package/classnames). you dont wanna include ternary operations when using this function

Reply:
since overlayTitle is optional we discussed that the ternary is ok.